src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java

```diff
@@ -173,4 +183,8 @@ public void setOrganisationIdentifier(String organisationIdentifier) {
     public boolean isOrganisationStatusActive() {
         return OrganisationStatus.ACTIVE == getStatus();
     }
+
+    public OrganisationMfaStatus getOrganisationMfaStatus() {
```

Comment:
I believe we are using `@Getter/@Setter` - is this required?

Reply:
all the other fields in this entity have explicit getters, so just tried to stay consistent but can remove it if required.

Reply:
I don't think it is required, remove and see if everything compiles fine - if not add back in